tests/purs/optimize/4386.purs

```diff
@@ -15,3 +15,11 @@ myInt1 = runSTFn1 mySTFn1 0
 
 myInt2 :: forall r. ST r Int
 myInt2 = runSTFn2 mySTFn2 0 1
+
+otherTest :: forall r. ST r Int
+otherTest = do
+  a <- runSTFn2 mySTFn2 0 1
+  b <- runSTFn1 mySTFn1 2
+  c <- myInt1
+  d <- myInt2
+  pure $ a + b + c + d
```

Comment:
Small nit, but could you add a blank newline to the end of this file?

Reply:
https://github.com/purescript/purescript/pull/4386/commits/73fac965e9406f7995899c51cdfc3755f1bf5e19

Reply:
Thanks!